Src/Plugins/Tests.Security/Helpers/HttpMockHelper.cs

```diff
@@ -18,12 +18,30 @@ namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Helpers
 {
     public class HttpMockHelper : DelegatingHandler
     {
-        private readonly List<Tuple<HttpRequestMessage, HttpStatusCode, HttpContent>> _fakeResponses =
-            new List<Tuple<HttpRequestMessage, HttpStatusCode, HttpContent>>();
+        public static HttpResponseMessage DefaultOkResponseMessage =
```

Comment:
>DefaultOkResponseMessage [](http://example.com/codeflow?start=42&length=24)

Can we please leave the casing of OK consistent with HttpStatus.OK?

i.e., DefaultOKResponseMessage.

Also, do we really require the 'Default' prefix? Isn't that implied by living in our helper?

Can we just use 

OKReponse?
NotFoundResponse, 

etc.?

Do you have other ideas.

Also, why aren't these declared as readonly?

Reply:
thanks for the suggestions.
OK/NotFound/...Response are great!